### ISSUE TITLE
Nefunkční aktualizace

### ISSUE THREAD

**Issue by zoidbergthepopularone** [fitak/fitak#82]
Už několik dní se neaktualizují příspěvky, je třeba obnovit Facebookový token.

**Comment by zoidbergthepopularone** [comment]
Opět vypršel token.

**Comment by tajo** [comment]
Tentokrat to neni token, ale chyba API: 

```
0 attempt per query (/343257012467087/feed).Caught exception: Unsupported get request. Object with ID '343257012467087' does not exist, cannot be loaded due to missing permissions, or does not support this operation. Please read the Graph API documentation at https://developers.facebook.com/docs/graph-api
```

@hermamat jakt to vypada s tvoji verzi?

**Comment by tajo** [comment]
Ja uz nemam motivaci se v tom stourat (uz jsem pryc z FITu nekolik let). Pokud se toho nechopi nekdo jiny, tak tohle je game over.

// Nemam vsak problem i nadale server a domenu provozovat.

**Comment by Mikulas** [comment]
Tohle je ten nejmenší problém, horší je, že v facebook pomalu odrotovává verzi api, kterou potřebujeme. V nových verzích už nejde jako uživatel tahat všechny příspěvky ze skupiny.

**Comment by hermamat** [comment]
@tajo No, s mojí verzí je to špatné. Mám toho teď až nad hlavu a vůbec nevím, co dřív.
Každopádně do obhajoby by mělo být alespoň "něco" na webu, takže na tom ještě budu muset chvíli zapracovat, takže něco bude, ale určitě to hned nebude 100 % funkční a nasaditelné...

Ale určitě bych to chtěl dotáhnout, když už jsem na tom dělal tak dlouho..ale nějak není vůbec čas :(

**Comment by tajo** [comment]
@Mikulas Neni to uz prave duvod, proc ted prestal crawler fungovat? (token je porad platny)

**Comment by Mikulas** [comment]
Rozhodně je to možný. Zkoušim teď udělat nějaký nový crawler proti nové verzi fb kompatibilní s live fiťákem…

**Comment by Mikulas** [comment]
@tajo mam testovací verzi v go, která docela funguje. Teď na hotfix by ale bylo nejlepší řešení
- [x] v `fbparams.neon` změnit permissions na `user_managed_groups` – v repu nejsou produkční configy, takže to nemůžu deploynout. Změníš to prosím? A nevím za koho se tam generují tokeny, musí to být teď pod [FitakBot](https://www.facebook.com/fitakbot?fref=ts).
- [ ] Dále je potřeba, aby se [FitakBot](https://www.facebook.com/fitakbot?fref=ts) stal adminem všech skupin, které má parsovat. To si beru na starost já.

**Comment by tajo** [comment]
@Mikulas Hotovo. FitakBot se uz pouzival. Regeneroval jsem token a odklepnul nove permissions.

**Comment by Mikulas** [comment]
Moc se to nepohnulo, takže jsem dneska poslal zprávu všem adminům podle https://docs.google.com/spreadsheets/d/1LgjauqEOJTkyRFpzunr4FAosTHKgu97zdjoDoIUS7OI/edit?usp=sharing
